filament/src/materials/skybox.mat

```diff
@@ -145,5 +171,6 @@ vertex {
         float3 p = getPosition().xyz;
         float3 unprojected = mulMat4x4Float3(getViewFromClipMatrix(), p).xyz;
         material.eyeDirection.xyz = mulMat3x3Float3(getWorldFromViewMatrix(), unprojected);
+        material.clipPos.xy = p.xy;
```

Comment:
there's a `getNormalizedViewportCoord()` built-in available in the fragment shader